payments-core/src/main/java/com/stripe/android/model/parsers/DeferredSetupIntentJsonParser.kt

```diff
@@ -43,7 +42,7 @@ class DeferredSetupIntentJsonParser(
             paymentMethodId = null,
             created = timeProvider(),
             status = null,
-            usage = params.setupFutureUsage,
+            usage = StripeIntent.Usage.OffSession,
```

Comment:
It doesn't make sense to me for the client to decide this value or any other, since the client isn't creating the SetupIntent.  e.g. what if the merchant creates a SetupIntent with a different usage value?

I may be misunderstanding what this code does, though.

Reply:
Like, why aren't we looking at the json to determine this value?

Reply:
See https://github.com/stripe/stripe-android/pull/6614#discussion_r1180387484.